<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubScraperGUIClass</class>
 <widget class="QMainWindow" name="SubScraperGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SubScraperGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="selectLabel">
      <property name="text">
       <string>Select an existing profile or build a new one.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectButton">
      <property name="text">
       <string>Select existing profile</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buildButton">
      <property name="text">
       <string>Build new profile</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SubScraperGUI.qrc"/>
 </resources>
 <connections/>
</ui>
